packages/reassure-measure/src/config.ts

```diff
@@ -1,21 +1,32 @@
-import type { render as RntlRender } from '@testing-library/react-native';
+import type { render as RntlRender, cleanup as RntlCleanup } from '@testing-library/react-native';
+import type { render as RtlRender, cleanup as RtlCleanup } from '@testing-library/react';
 
 let render;
+let cleanup;
 
 try {
   // eslint-disable-next-line import/no-extraneous-dependencies
   render = require('@testing-library/react-native').render;
+  // eslint-disable-next-line import/no-extraneous-dependencies
+  cleanup = require('@testing-library/react-native').cleanup;
 } catch {
-  // TODO add web support
-  throw new Error('Reassure requires `@testing-library/react-native` to be installed');
+  try {
+    // eslint-disable-next-line import/no-extraneous-dependencies
+    render = require('@testing-library/react').render;
+    // eslint-disable-next-line import/no-extraneous-dependencies
+    cleanup = require('@testing-library/react').cleanup;
+  } catch {
+    throw new Error('Reassure requires `@testing-library/react-native` or ``@testing-library/react-` to be installed');
+  }
 }
 
 type Config = {
   runs: number;
   dropWorst: number;
   outputFile: string;
   verbose: boolean;
-  render: typeof RntlRender;
+  render: typeof RntlRender | typeof RtlRender;
+  cleanup: typeof RntlCleanup | typeof RtlCleanup;
```

Comment:
imo, cleanup could be make optional, as is not needed by us. Precisely it's really needed by RTL. (even RNTL should work fine)

Reply:
👍